README.md

```diff
@@ -74,7 +74,7 @@ instructions for other platforms, and just in general, we recommend you see [Ras
 
           # initialize pico-sdk from submodule
           # note: this must happen before project()
-          include(pico-sdk/pico_sdk_init.cmake)
+          include(pico-sdk/external/pico_sdk_init.cmake)
```

Comment:
I think this was correct as-is? https://github.com/raspberrypi/pico-sdk/blob/master/pico_sdk_init.cmake